src/sql/workbench/contrib/notebook/browser/models/notebookModel.ts

```diff
@@ -190,8 +188,8 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		return this._cells;
 	}
 
-	public get contexts(): IDefaultConnection {
-		return this._activeContexts;
+	public get context(): ConnectionProfile {
```

Comment:
>context [](start = 12, length = 7)

There's other functions in here that refer to contexts (plural) such as contextsChanged, would suggest updating those as well